check50/__main__.py

```diff
@@ -278,42 +307,47 @@ def main():
     global SLUG
     SLUG = args.slug
 
-
+    # dev implies offline and verbose "info" if not overwritten
     if args.dev:
         args.offline = True
-        args.verbose = True
+        if not args.verbose:
+            args.verbose = "info"
 
+    # offline implies local
     if args.offline:
         args.local = True
 
+    # Setting any verbosity level implies logs from checks
     if args.verbose:
-        # Show lib50 commands being run in verbose mode
-        logging.basicConfig(level=os.environ.get("CHECK50_LOGLEVEL", "INFO"))
-        lib50.ProgressBar.DISABLED = True
         args.log = True
 
+    # Setup logging for lib50 depending on verbosity level
+    setup_logging(args.verbose)
+
     # Filter out any duplicates from args.output
     seen_output = set()
     args.output = [output for output in args.output if not (output in seen_output or seen_output.add(output))]
 
     # Set excepthook
-    excepthook.verbose = args.verbose
+    excepthook.verbose = bool(args.verbose)
     excepthook.outputs = args.output
     excepthook.output_file = args.output_file
 
+    # If remote, push files to GitHub and await results
     if not args.local:
         commit_hash = lib50.push("check50", SLUG, internal.CONFIG_LOADER, data={"check50": True})[1]
         with lib50.ProgressBar("Waiting for results") if "ansi" in args.output else nullcontext():
             tag_hash, results = await_results(commit_hash, SLUG)
+    # Otherwise run checks locally
     else:
-        with lib50.ProgressBar("Checking") if not args.verbose and "ansi" in args.output else nullcontext():
```

Comment:
One minor inconsistency I noted was that we were always printing out the ProgressBars regardless of verbosity level in remote mode:

```
Connecting....
Authenticating...
...
```

But in `--local` or `--offline` mode with any verbosity level we would stop printing the `Checking...` ProgressBar. So I made that consistent. check50 now always prints the ProgressBars, but does not animate them (with more dots) with any set verbosity level.